app/javascript/react/components/Navbar/Navbar.tsx

```diff
@@ -30,68 +31,91 @@ const NavItem = ({
+          {t("navbar.sections.airBeam")}
         </NavItem>
         <S.SubNav>
-          <NavItem href={urls.airbeamUserStories}>User Stories</NavItem>
-          <NavItem href={urls.airbeamHowItWorks}>How it Works</NavItem>
-          <NavItem href={urls.airbeamFaq}>FAQ</NavItem>
-          <NavItem href={urls.airbeamUsersGuide}>User's Guide</NavItem>
-          <NavItem href={urls.airbeamBuy}>Buy it Now</NavItem>
+          <NavItem href={urls.airbeamUserStories}>
+            {t("navbar.sections.userStories")}
+          </NavItem>
+          <NavItem href={urls.airbeamHowItWorks}>
+            {t("navbar.sections.work")}
+          </NavItem>
+          <NavItem href={urls.airbeamFaq}>{t("navbar.sections.faq")}</NavItem>
+          <NavItem href={urls.airbeamUsersGuide}>
+            {t("navbar.sections.usersGuide")}
+          </NavItem>
+          <NavItem href={urls.airbeamBuy}>
+            {t("navbar.sections.buyNow")}
+          </NavItem>
         </S.SubNav>
         <NavItem isNavTitle href={urls.aircasting}>
-          AirCasting
+          {t("navbar.sections.airCasting")}
         </NavItem>
         <S.SubNav>
-          <NavItem href={urls.map}>AirCasting Maps</NavItem>
-          <NavItem href={urls.android}>Android App</NavItem>
-          <NavItem href={urls.iOS}>iOS App</NavItem>
-          <NavItem href={urls.actions}>AirCasting Actions</NavItem>
+          <NavItem href={urls.map}>{t("navbar.sections.maps")}</NavItem>
+          <NavItem href={urls.android}>
+            {t("navbar.sections.androidApp")}
+          </NavItem>
+          <NavItem href={urls.iOS}>{t("navbar.sections.iosApp")}</NavItem>
+          <NavItem href={urls.actions}>{t("navbar.sections.actions")}</NavItem>
         </S.SubNav>
         <NavItem isNavTitle href={urls.about}>
-          About HabitatMap
+          {t("navbar.sections.about")}
         </NavItem>
         <S.SubNav>
-          <NavItem href={urls.history}>History & People</NavItem>
-          <NavItem href={urls.press}>Press</NavItem>
+          <NavItem href={urls.history}>{t("navbar.sections.history")}</NavItem>
+          <NavItem href={urls.press}>{t("navbar.sections.press")}</NavItem>
         </S.SubNav>
         <NavItem isNavTitle isUnderline href={urls.blog}>
-          TakingSpace Blog
+          {t("navbar.sections.blog")}
         </NavItem>
         <S.BottomNavContainer>
           <a href={urls.search}>
-            <img src={serchIcon} alt="Search icon" aria-label="Open search" />
+            <img
+              src={serchIcon}
+              alt={t("navbar.altSearch")}
+              aria-label={t("navbar.altSearch")}
+            />
           </a>
-          <S.Link href={urls.donate}>Donate</S.Link>
-          <S.BuyCTA href={urls.airbeamBuy}>Get Airbeam</S.BuyCTA>
+          <S.Link href={urls.donate}>{t("navbar.sections.donate")}</S.Link>
+          <S.BuyCTA href={urls.airbeamBuy}>
+            {t("navbar.sections.getAirBeam")}
+          </S.BuyCTA>
```

Comment:
Maybe we can adjust, so the names are corresponding? Here I like `getAirbeam` better.